src/containers/Review/ReviewSubmit.tsx

```diff
@@ -92,23 +92,19 @@ const ReviewSubmitButton: React.FC<ReviewSubmitProps & ReviewSubmitButtonProps>
   }
 
   const showPendingReviewWarning = () => {
-    const { title, message, option } = messages.REVIEW_STATUS_PENDING
```

Comment:
I am a bit confuse about this changes in this file

Reply:
Yes, sorry that was a mistake. But didn't remember to revert. Reverted now: f67f7e3

An remaining change to cover the case - when trying to submit and review status is changed to LOCKED is added as part of showing different messages of successful or failed submission on issue #308.